robot_modeling/common/load_frc_logs.py

```diff
@@ -0,0 +1,24 @@
+
+import numpy as np
+
+def load_sysid_logs(log_filename, sysid_data):
+    with open(log_filename) as f:
+        log = f.read().split("\n")
+
+    for index, line in enumerate(log):
+        if line.strip() == "":
+            continue
+        try:
+            timestamp, id, data = line.split(",")
+        except Exception:
+            print(line)
+            continue
+        
+        try:
+            if id in set(sysid_data.keys()):
```

Comment:
that's a much better way to do that, we should open a PR for the wpilib_process_logs.py file to be better. Actually, I guess I can just open a PR for feat/better-log-processing

Reply:
for what it's worth: if the new logging framework goes in at some point, we won't need to run a script to process the logs - they'll already be in the CSV format we already use pretty much